Makefile

```diff
@@ -0,0 +1,23 @@
+all: backend frontend
```

Comment:
Czy to nie jest to samo, co `start`?

Reply:
W sensie można by zrezygnować ze `start`, jeżeli i tak zawsze używacie po prostu samego `make`, albo usunąć `all` - obecnie to jest duplikacja